doc/jsk_perception/nodes/craft_node.rst

```diff
@@ -0,0 +1,89 @@
+craft_node.py
+=============
+
+What is this?
+-------------
+
+.. image:: ./images/craft_node.jpg
+
+Detect text region from image.
+This is a ROS node for ``CRAFT: Character-Region Awareness For Text detection``.
+
+In order to use this feature, you need to install `pytorch <https://pytorch.org/get-started/locally/>`_ (pytorch >= 1.9.0 is recommended).
+However, python2 user is not supposed to be able to install that version of torch.
+For python2 users, download the appropriate wheel file in your environment from https://download.pytorch.org/whl/cu90/torch_stable.html (melodic) and install with pip as follow:
+
+.. code-block:: bash
+
+   pip install --user torch-1.1.0-cp27-cp27mu-linux_x86_64.whl
+   pip install --user torchvision-0.3.0-cp27-cp27mu-manylinux1_x86_64.whl
+
+
+Subscribing Topic
+-----------------
+
+* ``~input`` (``sensor_msgs/Image``)
+
+  Raw image.
+
+
+Publishing Topic
+----------------
+
+* ``~output/cluster_indices`` (``jsk_recognition_msgs/ClusterPointIndices``)
+
+  Image indices of each detected text region.
+
+* ``~output/rects`` (``jsk_recognition_msgs/RectArray``)
+
+  Rectangles of detected texts.
+
+* ``~output/polygons`` (``jsk_recognition_msgs/PolygonArray``)
+
+  Polygons of detected texts.
+  Unlike ``RectArray``, this is output as a list of points on a 2D plane.
+
+
+Parameters
+----------
+
+* ``~gpu`` (Int, default: ``0``)
+
+  GPU id.
```

Comment:
If a version that runs on the CPU is supported, it might be easier for users to understand when the value to set in the case of the CPU is written like https://github.com/jsk-ros-pkg/jsk_recognition/blame/master/doc/jsk_perception/nodes/hand_pose_estimation_2d.rst#L39-L41

Reply:
https://github.com/jsk-ros-pkg/jsk_recognition/pull/2650/commits/85a4d68db370afe014a1903919cbf9d37d5232da
Thanks. I added the information.

Reply:
Thank you!